_data/simple-icons.json

```diff
@@ -8066,7 +8066,7 @@
         {
             "title": "Semaphore CI",
             "hex": "19A974",
-            "source": "https://semaphoreci.com/"
+            "source": "https://wpblog.semaphoreci.com/wp-content/themes/semaphoreci/img/logo-1.svg"
```

Comment:
Can you keep https://semaphoreci.com/ for the source? We generally prefer links to pages over files to reduce the possibility of future "link rot".

Reply:
sure, of course.